data/outersys.ssc

```diff
@@ -235,6 +241,7 @@ ReferencePoint "Orcus-Vanth" "Sol"
 	Mesh	"asteroid.cms"
 	Texture	"asteroid.*"
 	Radius	50  # dynamical upper limit
+	Mass	

```

Comment:
Missing a value here; I'd also suggest adding comments with the source of these masses.

Reply:
Got it, I've uploaded updates to these. Xiangliu does not have mass value as far as I know so I simply removed it - I forgot to before.